app/globals.css

```
@@ -375,3 +375,558 @@ h6 {
+}
+
+/* Mobile Browser - Force Hide Address Bar WITH SCROLLING */
+@media screen and (max-width: 768px) {
+  html {
+    height: 100vh;
+    height: -webkit-fill-available;
+  }
+  
+  body {
+    min-height: 100vh;
+    min-height: -webkit-fill-available;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

min-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)